build.go

```diff
@@ -852,13 +851,16 @@ func getReleaseVersion() (string, error) {
 }
 
 func getGitVersion() (string, error) {
+	// The current version as Git sees it
 	v, err := runError("git", "describe", "--always", "--dirty")
 	if err != nil {
 		return "", err
 	}
+
+	versionRe := regexp.MustCompile(`-[0-9]{1,3}-g[0-9a-f]{5,10}`)
```

Comment:
Why does this var need to be local now?

Reply:
It doesn't need to, but it was just used in one place and having it nearer where it's used seemed clearer.